gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/DagProcUtils.java

```diff
@@ -191,15 +194,10 @@ public static void cancelDag(Dag<JobExecutionPlan> dag, DagManagementStateStore
     }
   }
 
-  private static void sendCancellationEvent(Dag.DagNode<JobExecutionPlan> dagNodeToCancel, Properties props)
-      throws ExecutionException, InterruptedException {
+  private static void sendCancellationEvent(Dag.DagNode<JobExecutionPlan> dagNodeToCancel) {
```

Comment:
`sendCancellationEvent` should only send cancellation event. the violation of this led to bugs in the previous commit